backend/tests/services/web_scraper/test_markdown_cleaner.py

```
@@ -0,0 +1,41 @@
+
+    assert "Home Pricing" not in cleaned
+    assert "Share" not in cleaned
+    assert "Copyright" not in cleaned
+    assert "This paragraph should stay." in cleaned
+
+
+def test_cleaner_keeps_body_text_with_action_words() -> None:
+    html = """
+    <article>
+      <h1>操作说明</h1>
+      <p>点击登录按钮后，表单会显示在页面右侧。</p>
```

Ruff (0.15.15)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)